formulas/mo/files/mo

```diff
@@ -75,7 +75,7 @@ mo() (
                         ;;
 
                     --source=*)
-                        f2source="${1#--source=}"
+                        f2source="${arg#--source=}"
```

Comment:
Yeah same question. It seems like the old way was just incorrect? I don't see any shift statements.